lib/install.sh

```diff
@@ -25,6 +28,17 @@ get_platform() {
     fi
 }
 
+# Get the platform name.
+get_arch() {
+    local platform_arch
+    platform_arch=$(uname -p)
```

Comment:
`uname -p` on the m1 comes back as `arm`. When we go to copy the file, we look for `phylum-macos-arm`, which doesn't exist. We do have `phylum-macos-aarch64` in the zip though. Should we rename the produced file to `phylum-macos-arm`?

Reply:
Updated to match against first 3 chars (i.e. `arm.*`), so it _should_ copy the right bin now.

Reply:
Ok cool, I'll re-test on an m1 here in a bit!

Reply:
Awesome thanks!